### ISSUE TITLE
Text editor for post creation

### ISSUE THREAD

**Issue by DimiMikadze** [DimiMikadze/orca#183]
It will be nice to give users the ability to format their content as they want. For example, make text bigger, bolder, etc., while creating or updating the post.

Sample UI can be found as a screenshot. We will not be supporting the emojis dropdown at this moment. This issue is about only formatting the text.

<img width="403" alt="Screen Shot 2021-11-14 at 9 40 44 AM" src="https://user-images.githubusercontent.com/8871615/141669110-334439fc-e462-4886-acb6-f3b2f6b68873.png">

I found these libraries for creating text editors with React:
- https://github.com/sstur/react-rte
- https://github.com/jpuri/react-draft-wysiwyg
- https://github.com/ianstormtaylor/slate

I'm open to discussing other options.

**Comment by RajendrasinhParmar** [comment]
@DimiMikadze Can we try going with https://github.com/facebook/draft-js which seems a good option?

And if not then I think slate will be the better option.

Let me know your inputs about the same. I can try to work on this one.

**Comment by DimiMikadze** [comment]
@RajendrasinhParmar thanks for asking. I think Slate is a more suited library for our purposes since we are not planning to build an advanced editor with custom features, we just need a basic editor with minimal features like the size and weight of the font. What do you think?

**Comment by RajendrasinhParmar** [comment]
@DimiMikadze Thank you for the update.
I will consider using the slate and will try to create a PR soon.

**Comment by RajendrasinhParmar** [comment]
@DimiMikadze Hello, I've started working on this issue and I've almost done with the implementation of the text editor. And now I'm working on actually saving the response in the database and displaying the rich text on the front-end posts.

**Comment by faerylay** [comment]
@RajendrasinhParmar  do you have any repo for this project too? I'm junior developer, I try to do social media app like this one. I'm using express session and material UI and I had some issue with subscription(like user online status and Notifications for real time update) . And some other...

**Comment by RajendrasinhParmar** [comment]
@faerylay 
I'm contributing to this project from fork of the same project. 
this is the fork that I'm using for contribution to this project.

**Comment by faerylay** [comment]
@RajendrasinhParmar Thanks u I found it but you're using rest API instead of graphql

**Comment by RajendrasinhParmar** [comment]
@faerylay that's correct. However, I'm one of the contributors to this project and trying to contribute in any way possible. So the project itself is using REST APIs so will be using the REST APIs as of now.
